Samples.xcodeproj/project.pbxproj

```diff
@@ -1100,7 +1114,7 @@
 			repositoryURL = "https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/";
 			requirement = {
 				kind = upToNextMinorVersion;
-				minimumVersion = 200.0.0-beta;
+				minimumVersion = "200.0.0-beta";
```

Comment:
This change can be reverted.

Reply:
```suggestion
				minimumVersion = 200.0.0-beta;
```

Reply:
I reverted that change, but Xcode keeps making it, so you might want to go with it.